### ISSUE TITLE
MacOS Mojave 10.14(Beta) GUI Crashed

### ISSUE THREAD

**Issue by Calebe94** [dylanaraps/pywal#292]
I've installed pywal in my MacOS Mojave following the [Installation Guide](https://github.com/dylanaraps/pywal/wiki/Installation) and after that I've run: 

`wal -i path`

and then my MacOS GUI crashed, turning everything black. Now I can't maximize anything, can't use any gestures from trackpad and my Launchpad, Mission Control and Dock doesn't open anymore.

![](https://i.imgur.com/IHqh9Vp.png)

I've created another user to see if my MacOS was broken, but everything was OK.

TL:DR : Ran the wal command and it broke my default user, making me create a new one.

Hope that you could help me, or it might help someone else.

**Comment by danruto** [comment]
You can delete `Library/Application Support/Dock/desktoppicture.db` or go through it to find the bad settings. 

Note if you do delete it, it'll obviously reset your settings but better than having nothing working :)

**Comment by mTvare6** [comment]
@Calebe94 maybe I am too late, but what all are the settings it resets?

**Comment by Calebe94** [comment]
@mTvare6 as far as I remember it was the main UI things that stop to work such as the Dock and Mission Control. Even when I tried to run from the `command+space` thing (can't remember the name). Then I've created another user to check if it was a system wide problem, but it was not. And @danruto gave me the solution. 

This is everything I can remember.
